src/pudl/transform/params/ferc1.py

```diff
@@ -3448,6 +3448,119 @@
             "on": "ferc_account_label",
         },
     },
+    "electric_operating_revenues_ferc1": {
+        "rename_columns_ferc1": {
+            "duration_xbrl": {
+                "columns": {
+                    f"megawatt_hours_sold_{col}": f"{col}_mwh_sold"
+                    for col in [
+                        "residential_sales",
+                        "small_or_commercial",
+                        "large_or_industrial",
+                        "public_street_and_highway_lighting",
+                        "other_sales_to_public_authorities",
+                        "sales_to_railroads_and_railways",
+                        "interdepartmental_sales",
+                        "sales_to_ultimate_consumers",
+                        "sales_for_resale",
+                        "sales_of_electricity",
+                        "provision_for_rate_refunds",
+                        "revenues_net_of_provision_for_refunds",
+                    ]
+                }
+                | {
+                    f"average_number_of_customers_per_month_{col}": f"{col}_avg_customers_per_month"
+                    for col in [
+                        "residential_sales",
+                        "small_or_commercial",
+                        "large_or_industrial",
+                        "public_street_and_highway_lighting",
+                        "other_sales_to_public_authorities",
+                        "sales_to_railroads_and_railways",
+                        "interdepartmental_sales",
+                        "sales_to_ultimate_consumers",
+                        "sales_for_resale",
+                        "sales_of_electricity",
+                        "provision_for_rate_refunds",
+                        "revenues_net_of_provision_for_refunds",
+                    ]
+                }
+                | {
+                    col: f"{col}_electric_operating_revenue"
+                    for col in [
+                        "residential_sales",
+                        # Note: the small_or_commercial/large_or_industrial columns are already prefixed
```

Comment:
Not sure why, but these two columns have the prefix `electric_operating_revenue` already, which is a bit inconsistent with the rest of the columns

Reply:
lol. that is weird but also is consistent with FERCs inconsistency.